Index: PythonQt.pro
===================================================================
--- PythonQt.pro	(revision 403)
+++ PythonQt.pro	(working copy)
@@ -1,4 +1,4 @@
 TEMPLATE = subdirs
 
 CONFIG += ordered
-SUBDIRS = generator src extensions tests examples
+SUBDIRS = src
Index: build/python.prf
===================================================================
--- build/python.prf	(revision 403)
+++ build/python.prf	(working copy)
@@ -2,12 +2,12 @@
 
 # Change this variable to your python version (2.6, 2.7, 3.3, ...)
 win32:PYTHON_VERSION=26
-unix:PYTHON_VERSION=2.6
+unix:PYTHON_VERSION=$$system(python3 -c \'import sys; print(\".\".join(map(str, sys.version_info[:2])))\')
 
 macx {
   # for macx you need to have the Python development kit installed as framework
-  INCLUDEPATH += /System/Library/Frameworks/Python.framework/Headers
-  LIBS += -F/System/Library/Frameworks -framework Python
+  INCLUDEPATH += $$system(python$${PYTHON_VERSION}-config --prefix)/Headers
+  LIBS += -L$$system(python$${PYTHON_VERSION}-config --prefix)/lib $$system(python$${PYTHON_VERSION}-config --libs)
 } else:win32 {
   # for windows install a Python development kit or build Python yourself from the sources
   # Make sure that you set the environment variable PYTHON_PATH to point to your
@@ -25,9 +25,9 @@
 
   # check if debug or release
   CONFIG(debug, debug|release) {
-    DEBUG_EXT = _d 
+    DEBUG_EXT = _d
   } else {
-    DEBUG_EXT = 
+    DEBUG_EXT =
   }
 
   win32:INCLUDEPATH += $$(PYTHON_PATH)/PC $$(PYTHON_PATH)/include
@@ -35,7 +35,7 @@
 } else:unix {
   # on linux, python-config is used to autodetect Python.
   # make sure that you have installed a matching python-dev package.
-  
+
   unix:LIBS += $$system(python$${PYTHON_VERSION}-config --libs)
   unix:QMAKE_CXXFLAGS += $$system(python$${PYTHON_VERSION}-config --includes)
 }
Index: src/PythonQt.h
===================================================================
--- src/PythonQt.h	(revision 403)
+++ src/PythonQt.h	(working copy)
@@ -42,6 +42,7 @@
 */
 //----------------------------------------------------------------------------------
 
+#include <iostream>
 #include "PythonQtUtils.h"
 #include "PythonQtSystem.h"
 #include "PythonQtInstanceWrapper.h"
@@ -56,7 +57,6 @@
 #include <QByteArray>
 #include <QStringList>
 #include <QtDebug>
-#include <iostream>
 
 
 class PythonQtClassInfo;
